### PR TITLE
misc(proto): backport proto formatting fixes

### PR DESCRIPTION
**Pull request by paulirish**
In LR, the linter often complains about the formatting of this file. I'm backporting the automatic formatting changes that satisfy the linter.

Technically the linter didn't like the 5 instances of trailing whitespace, and the format checker didn't like the non-wrapping text.

So anyway.. both fixed.

-------

and no i don't see a simple way we can compatibly format out here in GH when landing  changes to the file. :/

### CODE REVIEW

File: proto/lighthouse-result.proto [hunk 1]
```diff
@@ -131,7 +132,8 @@ message LighthouseResult {
     // This enum served the emulated_form_factor field, but in v7, that field
     // was deprecated. Meanwhile in v7, the form_factor field was added and has
     // a mobile/desktop enum, so we simplify reuse, minus the `none` option.
-    // See https://github.com/GoogleChrome/lighthouse/blob/master/docs/emulation.md#changes-made-in-v7
+    // See
+    // https://github.com/GoogleChrome/lighthouse/blob/master/docs/emulation.md#changes-made-in-v7
```
Comment (patrickhulce) [inline proto/lighthouse-result.proto:136]: image

Comment (patrickhulce) [reply proto/lighthouse-result.proto:136]: seriously though, how do all the internal google formatters that wind up touching lh/chrome-launcher have such bad wrapping rules? 😭 

clang-format and the markdown reformatter for devsite were similarly awful on the line wrap stuff.

Comment (paulirish) [reply proto/lighthouse-result.proto:136]: _sometimes_, the real opensource world is better than internal google tools. 😛

**Review by adamraine — APPROVED** [review]
Thank you @paulirish, very cool!

**Pull request merged**